sys/shell/cmds/cord_ep.c

```diff
@@ -51,23 +52,48 @@ static int _cord_ep_handler(int argc, char **argv)
     int res;
 
     if ((argc > 1) && (strcmp(argv[1], "register") == 0)) {
-        char *regif = NULL;
         if (argc < 3) {
-            printf("usage: %s register <server address> [registration interface]\n",
+            printf("usage: %s register <registration uri>\n",
                    argv[0]);
+            printf("example: %s register coap://[2001:DB8::1]:99/login\n", argv[0]);
```

Comment:
```suggestion
            printf("If the registration URI's path is empty, the registration resource is auto-discovered.\n"]);
            printf("example: %s register coap://[2001:db8::1]:99\n", argv[0]);
```

I think that not having the path and using autodiscovery should be the normal case, and advertised as such.

(And adjusting capitalization to match RFC5952)

Reply:
Agreed.